Coin dd

BirgitSchneider opened the pull request:
Hello all, 

I added the macros in cables/*/*/coin/log 
The others will follow in in the next days.

Cheers, 
Birgit

vonsturm commented:
... ok let's wait until all the macros are there. Anyways I would simulate this at mpik because the transfer needs a lot of time.

BirgitSchneider commented:
Hi all, 
I updated the coin info in the README.md and added the macros where the statistics are already checked for:
cables
ge_holders
gedet/lar_pplus
gedet_nplus
larveto/copper_shroud
larveto/sipm_ring

still missing & following in the next days: 
gedet/nplus
minishroud

I would copy the root and out files to MPIK and let you know, when the stuff is there. It is at the moment about 500 GB in total.

Cheers, 
Birgit

BirgitSchneider commented:
Updated all macros, despite of checked statistics.
